### PR TITLE
Add Meeting Artifacts 5 Sept 2024

### PR DESCRIPTION
**Pull request by riaankleinhans**
@renuchauhan221

### CODE REVIEW

**Review by AlecFernandez — APPROVED** [review]

File: TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+* TikTok - Mingshen Sun +  / Yao Zhang
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Alexander Manilov
+* Andey Pogoreltev (Super Protocol)
+* Chandra Nelogal (Self)
+* Diona Glaze
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md:54]:
```suggestion
* Diona Glaze (Google)
```

File: TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+Dan Middleton (DM) opened the call at 7:04 am PT and reviewed the antitrust policy. 
+
+## Agenda
+
+DM reviewed the agenda for the meeting.
+
+## Roll Call / Attendance
+
+Quorum was established as no votes were required.
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md:87]:
```suggestion
Quorum was established although no votes were required.
```

File: TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+Mike Bursell (MB) introduced Attestation & AVS (Attestation Verification Services) and discussed the topics that came out of the Attestations Business models document. He discusses the way CCC should talk about attestation and remote attestation. He proposed that definitions need to be calcified. Chandra Nelogal (CN) agreed that the term remote could mean several different things and continued to explain his perspective on the topic. Nathaniel McCallum (NM) said that Attestation is becoming a loaded term with different definitions in various industries. Further discussion ensued.
+
+DM suggested that the work should be broken down into work packages that could be addressed through working groups. Catherine Zhang ask if the work packages would be addressed by existing working groups or new groups. MB proposed that a portion of the TAC meeting could be set apart for that on a bi-weekly basis. DM proposed that the TAC could assign questions and work to the existing working groups to be addressed. DM asked meeting participants to communicate on the topic through the TAC Slack channel and mailing list and it would be further discussed in the next meeting.
+
+## Super Protoco Tech Talk
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md:111]:
```suggestion
## Super Protocol Tech Talk
```

File: TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+Nukri Basharuli (Founder/CEO) introduced themselves and the Super Protocol team. Nukri and Andrey Pogoreltsev - CTO of Super Protocol continued to present their tech talk. [Presentation](./CCC_Super%20Protocol%20Tech%20overview_Sept%205'24.pdf)
+
+##  2024 CCC TAC Objectives
+
+DM introduced the TAC goals document. Yash Mankad updated the meeting on the progress on the Mentorship program and thanked SK for driving participation from the CCC projects. They anticipate receiving two project proposals soon from the Veriosin project and are discussing with Coconut SVSM. Sal Kimmich (SK) said the Veriason would be mentoring two mentees. Riaan would help determine the administrative requirements of one mentor working with two mentees.
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md:117]:
```suggestion
DM introduced the TAC goals document. Yash Mankad updated the meeting on the progress on the Mentorship program and thanked Sal Kimmich (SK) for driving participation from the CCC projects. They anticipate receiving two project proposals soon from the Veraison project and are discussing with Coconut SVSM. SK said the Veraison would be mentoring two mentees. Riaan will help determine the administrative requirements of one mentor working with two mentees.
```

**Review by dcmiddle — COMMENTED** [review]
> TAC/Meetings/2024/2024-09-05/CCC_Super Protocol Tech overview_Sept 5'24.pdf

Quotes in file names can cause problems. Can we rename the file...

TAC/Meetings/2024/2024-09-05/CCC_SuperProtocolTechOverview_2024-09-05.pdf

**Comment by riaankleinhans** [comment]
> > TAC/Meetings/2024/2024-09-05/CCC_Super Protocol Tech overview_Sept 5'24.pdf
> 
> Quotes in file names can cause problems. Can we rename the file...
> 
> TAC/Meetings/2024/2024-09-05/CCC_SuperProtocolTechOverview_2024-09-05.pdf

Good point, updated. Thanks

File: TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+DM suggested that the work should be broken down into work packages that could be addressed through working groups. Catherine Zhang ask if the work packages would be addressed by existing working groups or new groups. MB proposed that a portion of the TAC meeting could be set apart for that on a bi-weekly basis. DM proposed that the TAC could assign questions and work to the existing working groups to be addressed. DM asked meeting participants to communicate on the topic through the TAC Slack channel and mailing list and it would be further discussed in the next meeting.
+
+## Super Protocol Tech Talk
+
+Nukri Basharuli (Founder/CEO) introduced themselves and the Super Protocol team. Nukri and Andrey Pogoreltsev - CTO of Super Protocol continued to present their tech talk. [Presentation](./CCC_Super%20Protocol%20Tech%20overview_Sept%205'24.pdf)
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md:113]: update link

Comment (riaankleinhans) [reply TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md]: Good catch. Done.

**Review by dcmiddle — APPROVED** [review]

**Pull request merged**